cmake/onnxruntime_java.cmake

```diff
@@ -94,6 +94,8 @@ if(APPLE)
    endif()
    if(JNI_ARCH STREQUAL "x86_64")
        set(JNI_ARCH x64)
+   elseif(JNI_ARCH STREQUAL "arm64")
```

Comment:
What does the new package look like?
I suppose you will have "linux-aarch64" and "osx-aarch64"

Reply:
BTW, in nuget packages, both of them should be "arm64" instead of "aarch64".